service/src/main/java/bio/terra/pipelines/app/controller/JobsApiController.java

```diff
@@ -62,18 +62,20 @@ public ResponseEntity<ApiPostJobResponse> createJob(
     final SamUser userRequest = getAuthenticatedInfo();
     String userId = userRequest.getSubjectId();
     String pipelineVersion = body.getPipelineVersion();
+    Object pipelineInputs = body.getPipelineInputs();
```

Comment:
I think you probably want to make changes to the tests in JobsApiControllerTest.java - since the new "pipeline inputs" input should be required